common/src/main/java/org/keycloak/common/util/ServerCookie.java

```diff
@@ -27,7 +27,11 @@
 
 /**
  * Server-side cookie representation.  borrowed from Tomcat.
+ *
+ * @deprecated Should not be used on the Keycloak server-side, or in extensions. Will be removed when no longer used by
+ * adapters
  */
+@Deprecated
```

Comment:
The deprecation of `ServerCookie` is not mentioned in the upgrade docs.

Reply:
Added